pkg/controllers/flannelmigration/network_migrator.go

```diff
@@ -135,8 +135,8 @@ func (m *networkMigrator) checkCalicoVxlan(node *v1.Node) error {
 }
 
 // Drain node, remove Flannel and setup Calico network for a node.
-func (m *networkMigrator) setupCalicoNetworkForNode(node *v1.Node) error {
-	log.Infof("Setting node label to disable Flannel daemonset pod on %s.", node.Name)
+func (m *networkMigrator) setupCalicoNetworkForNode(node *v1.Node, index int) error {
+	log.Infof("Setting node label to disable Flannel daemonset pod on node %s[index %d].", node.Name, index)
```

Comment:
Same for this - can we move the log into the loop below so that we don't need to pass the index just for logging purposes?